src/bun.js/bindings/bindings.zig

```diff
@@ -873,6 +878,11 @@ pub const ZigString = extern struct {
 
     pub fn toExternalValue(this: *const ZigString, global: *JSGlobalObject) JSValue {
         this.assertGlobal();
+        if (this.len > String.max_length) {
+            bun.default_allocator.free(@constCast(this.byteSlice()));
```

Comment:
same here

Reply:
it is guaranteed. note how it never passes an allocator